src/core/datatablemodule.cc

```diff
@@ -186,17 +186,17 @@ R"(frame_integrity_check(frame)
 
 This function performs a range of tests on the `frame` to verify
 that its internal state is consistent. It returns None on success,
-or throws an AssertionError if any problems were found.
+or throws an :exc:`AssertionError` if any problems were found.
 
 Parameters
 ----------
 frame: Frame
-    A :class:`Frame` that needs to be checked for internal consistency.
+    A :class:`Frame` object that needs to be checked for internal consistency.
 
 return: None
 
 except: AssertionError
-    An error is raised if there were any issues with the `frame`.
+    An exception is raised if there were any issues with the `frame`.
```

Comment:
It seems that in other places we're saying "the exception", probably meaning that, like in this case, it is the `AssertionError` exception. But at the same time it is "an exception", because it is just one of the exceptions. Not sure what is better.

Reply:
I would leave "an" here, because there could really be over a dozen different exceptions here (though all of them are AssertionErrors).